src/musica_aerosol_modal.meta

```diff
@@ -0,0 +1,56 @@
+[ccpp-table-properties]
+  name = aerosol_modal_t
+  type = ddt
+  dependencies = musica_aerosol.F90,musica_wavelength_grid.F90
+
+[ccpp-arg-table]
+  name  = aerosol_modal_t
+  type  = ddt
+[ number_of_modes_ ]
+  standard_name = number_of_aerosol_modes
+  units = count
+  type = integer
+  dimensions = ()
```

Comment:
Would it pose any problems if `number_of_modes_` were private? I'm guessing not, because `state_` is not included in the meta data?